chef/cookbooks/provisioner/templates/suse/crowbar_register.erb

```diff
@@ -283,6 +296,18 @@ curl -L -o /etc/chef/validation.pem \
     --connect-timeout 60 -s \
     "$HTTP_SERVER/validation.pem"
 
+nodes=$(get_nodes_allow_fail || echo "")
+if test -z "$nodes"; then
+  echo "Failed to list nodes known on the administration server."
+  echo "Is the administration server up?"
+  exit 1
+fi
+
+if echo "$nodes" | grep -q "\"name\":\"$HOSTNAME\""; then
```

Comment:
@vuntz the url I proposed always works.. it will either return a json blob or a (html) error page. if it doesn't return anything the chef server is unreachable..